domehunter/dome_control.py

```diff
@@ -245,7 +246,11 @@ def __init__(self,
     @property
     def dome_az(self):
         """ """
-        return self._ticks_to_az(self._encoder_count)
+        if self._dome_az is None:
+            print("Cannot return Azimuth as Dome is not yet calibrated.\
+                   Run calibration loop")
+            return None
```

Comment:
This line is redundant, as the `return self._dome_az` after the `if` block` will also return `None` in this cae.

Reply:
Ah, yes. This was a relic of me disentangling the logging code. Originally I had
```diff
-        return self._ticks_to_az(self._encoder_count)
+        if self._dome_az is None:
+            logger.info("Cannot return Azimuth as Dome is not yet calibrated.\
+                   Run calibration loop")
+            return None
+        logger.debug(f'Dome azimuth reading: {self._dome_az}')
+        return self._dome_az
```

and I didn't want it reporting after the calibration error. But I think you are right that it can all be reworked anyway to log accurately and give accurate warning, thanks..